fnet/data/__init__.py

```diff
@@ -1,6 +1,7 @@
-from fnet.data.bufferedpatchdataset import BufferedPatchDataset
-from fnet.data.tiffdataset import TiffDataset
-from fnet.data.fnetdataset import FnetDataset
+from .bufferedpatchdataset import BufferedPatchDataset
```

Comment:
Why were all imports changed to be relative? According to [this](https://www.python.org/dev/peps/pep-0008/#imports), absolute imports are recommended. Also, if we wanted to change this particular `__init__.py` file, then we should change *all* of them.